tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -530,6 +538,14 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
         [ConfigKey(ConfigurationKeys.TraceEnabled)]
         internal bool TraceEnabledInternal { get; private set; }
 
+        /// <summary>
+        /// Gets a value indicating whether Appsec standalone is enabled.
+        /// Default is <c>false</c>.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.ExperimentalAppsecStandaloneEnabled"/>
+        [ConfigKey(ConfigurationKeys.ExperimentalAppsecStandaloneEnabled)]
```

Comment:
You don't need (and shouldn't add) the `[ConfigKey]` attribute. That's only necessary if it's part of the public API, and is used by various other bits, so if you remove this, you can remove all those other changes 🙂 

```suggestion
```

Reply:
:til: